### ISSUE TITLE
About the medmnistv2

### ISSUE THREAD

**Issue by 1170300714** [function2-llx/PUMIT#4]
Hello, thanks for your great work! 

It seems that the 'medmnistv2' is not in the scripts folder. Could you upload? Thanks!

**Comment by function2-llx** [comment]
Hi, thank you for your interest here! Unfortunately, recently our members are busy with other stuffs and we're unable to release this part of code very soon. We anticipate that we can release this part of code in a few weeks.

**Comment by 1170300714** [comment]
Thanks for reply!

Can you briefly introduce the key details about Classification Experiments? I have the following questions to ask you:
1. What is the specific implementation of the fully connect layer following ViT?
2. When finetune, should we only train the fully connected layer or finetune the entire model?
3. It has been observed in Table 5 of your paper's appendix that the input size of finetune is inconsistent with the input size of pretraining, which seems to lead to the pos_embed cannot be applied to finetune process. How can I solve this problem?
4. Regarding learning rate, Table 5 of the Appendix mentions that there are 9 initial learning rates, and I do not understand which one should be used for "Ours"
I am very grateful if you could show me more training details. Looking forward to your reply, thank you again!

**Comment by function2-llx** [comment]
@1170300714 Hi, we've uploaded the code for medmnistv2 now (you should switch to the `submit` branch to check it). 
For your question:
1. It's a linear layer, as you can find in the code: https://github.com/function2-llx/PUMIT/blob/submit/downstream/medmnistv2/eval-medmnist.py#L152
2. The entire model is fine-tuned.
3. We interpolate the pre-trained positional embedding to size used in fine-tuning, as you can find here: https://github.com/function2-llx/PUMIT/blob/submit/pumit/model/vit.py#L280
4. For each model, we experiment with all learning rates and report the best one.

I hope the code and the reply above address your questions.

**Comment by 1170300714** [comment]
收到，感谢您的来件

**Comment by 1170300714** [comment]
Thanks  for your contribution and release~
